examples/meta/src/neural_nets/convolutional_net_classification.sg

```diff
@@ -1,7 +1,7 @@
-File f_feats_train = csv_file("../../data/fm_train_mnist_16x16.dat")
-File f_feats_test = csv_file("../../data/fm_test_mnist_16x16.dat")
-File f_labels_train = csv_file("../../data/labels_train_mnist_16x16.dat")
-File f_labels_test = csv_file("../../data/labels_test_mnist_16x16.dat")
+File f_feats_train = csv_file("../../data/classifier_3class_256d_linear_features_train.dat")
```

Comment:
I am confused by the "linear" in the filename. 
Also I think it should still say "mnist"
What about `mnist_3class_256d_features_train.dat`

Reply:
yes that is a better name :)